src/main/java/org/snomed/snowstorm/rest/ControllerHelper.java

```diff
@@ -133,6 +135,15 @@ public static PageRequest getPageRequest(int offset, int limit, Sort sort) {
 		return sort == null ? PageRequest.of(page, size) : PageRequest.of(page, size, sort);
 	}
 
+	public static PageRequest getPageRequest(int offset, int limit, Sort sort, String searchAfter) {
+		validatePageSize(offset, limit);
+		if (!Strings.isNullOrEmpty(searchAfter)) {
+			return SearchAfterPageRequest.of(SearchAfterHelper.fromSearchAfterToken(searchAfter), limit, sort);
+		} else {
+			return ControllerHelper.getPageRequest(offset, limit);
+		}
+	}
+
```

Comment:
searchAfter here is nullable, which allows for a minor change in the controller.